ChangeLog.txt

```diff
@@ -12,6 +12,7 @@ MINOR:
 - Made the Pope the religious head of the Chalcedonian church
 
 MAP/HISTORY:
+- Added historical rulers of the Irish Petty Kingdoms up to year 600 along with their families
```

Comment:
Now that England is done, you could update the changelog with what you did there and in Scotland.